### PR TITLE
Add #nguyenvangiang line to readme.md file

### PR DESCRIPTION
**Pull request by giangnv-0040**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # pp1906_project2
 Laravel 6 project
+#nguyenvangiang
```
Comment (andt95) [inline README.md]: okok

Comment (giangnv-0040) [inline README.md]: ok

Comment (giangnv-0040) [reply README.md]: 111

File: README.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
 # pp1906_project2
 Laravel 6 project
```
Comment (namuetn) [inline README.md:2]: Nice

File: README.md (hunk 1 above)
Comment (giangnv-0040) [inline README.md]:
```suggestion
#nguyenvangiang19
```

Comment (andt95) [reply README.md]:
```suggestion
#dangthuaan
```
ok

File: README.md (hunk 2 above)
Comment (namuetn) [inline README.md:2]:
```suggestion
Laravel 6 project19
```

**Review by giangnv-0040 — COMMENTED** [review]
OK

**Review by namuetn — APPROVED** [review]
Okee

**Review by dangthuaan — APPROVED** [review]

**Pull request merged**